### ISSUE TITLE
[Suggestion] A 'collapse all' button for the file tree

### ISSUE THREAD

**Issue by joakim-m** [adobe/brackets#11205]
Title says all.

**Comment by redmunds** [comment]
@joakim-m Use `Ctrl/Cmd-click` on folder expand/collapse arrows in file tree to expand all/collapse all under folder. `Ctrl/Cmd-click` on root-level folder will expand/collapse entire tree.

Keeping this open so documentation gets added to [How to Use Brackets](https://github.com/adobe/brackets/wiki/How-to-Use-Brackets).

**Comment by joakim-m** [comment]
If you ctrl-click the sub-folders don't get collapsed, only the one you click on.

**Comment by MarcelGerber** [comment]
It wouldn't hurt, though, if we added that same functionality to the context menu (?). It could also mention the "shortcut" so it's more explorable.

**Comment by softomatix** [comment]
> @joakim-m Use Ctrl/Cmd-click on folder expand/collapse arrows in file tree to expand all/collapse all under folder. Ctrl/Cmd-click on root-level folder will expand/collapse entire tree.

Ctrl + Alt + Click?

**Comment by redmunds** [comment]
@softomatix Are you asking what I meant by `Ctrl/Cmd-click` ? If so, I meant `Cmd-click` on Mac and `Ctrl-click` on Windows and Linux.

**Comment by softomatix** [comment]
@redmunds Yes, I got it but it does not seem to work without `Alt` in combination. Just wanted to confirm that it is correct.

**Comment by redmunds** [comment]
@softomatix `Ctrl+Alt+click` expands/collapses the tree under the arrow that you click on. `Ctrl+click` expands/collapses _all_ trees in file tree.

That's how it works for me on Win7. Are you seeing something different? If so, what OS/version?

**Comment by softomatix** [comment]
@redmunds I can see the difference now. `Ctrl` + `Click` collapse trees on the same level but in fact it is not recursive. Once you expand any of those trees they will have all previously expanded trees expanded. Anyway, I've got your point. Thanks!
